Orange/ensembles/ada_boost.py

```diff
@@ -1,24 +1,37 @@
 import sklearn.ensemble as skl_ensemble
-from Orange.base import SklLearner
-from Orange.classification.base_classification import (SklLearnerClassification,
-                                                       SklModelClassification)
-from Orange.regression.base_regression import (SklLearnerRegression,
-                                               SklModelRegression)
 
-__all__ = ["SklAdaBoostLearner", "SklAdaBoostRegressionLearner"]
+from Orange.base import SklLearner, SklModel
+from Orange.classification.base_classification import (
+    SklLearnerClassification, SklModelClassification
+)
+from Orange.modelling import Fitter
+from Orange.regression.base_regression import (
+    SklLearnerRegression, SklModelRegression
+)
+
+__all__ = [
+    'SklAdaBoostLearner',
+    'SklAdaBoosClassificationLearner',
+    'SklAdaBoostRegressionLearner',
+]
 
 
 class SklAdaBoostClassifier(SklModelClassification):
     pass
 
 
-class SklAdaBoostLearner(SklLearnerClassification):
+class SklAdaBoosClassificationLearner(SklLearnerClassification):
     __wraps__ = skl_ensemble.AdaBoostClassifier
     __returns__ = SklAdaBoostClassifier
-    name = 'skl adaBoost'
+    name = 'skl adaBoost classification'
```

Comment:
Setting `.name` is somewhat redundant since https://github.com/biolab/orange3/blob/40bbcde23b32c5c61f29052bd381876a4f3dfef8/Orange/base.py#L144-L161